test/distributed/org/apache/cassandra/distributed/test/HintedHandoffNodetoolTest.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed.test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import org.apache.cassandra.config.DatabaseDescriptor;
+import org.apache.cassandra.distributed.Cluster;
+import org.apache.cassandra.hints.HintsService;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(Parameterized.class)
+public class HintedHandoffNodetoolTest extends TestBaseImpl
+{
+    private static Cluster cluster;
+
+    @Parameterized.Parameter
+    public int node;
+
+    @Parameterized.Parameters(name = "node={0}")
+    public static List<Object[]> data()
+    {   
+        List<Object[]> result = new ArrayList<>();
+        result.add(new Object[]{ 1 });
+        result.add(new Object[]{ 2 });
+        return result;
+    }
+
+    @BeforeClass
+    public static void before() throws IOException
+    {
+        cluster = init(Cluster.build().withNodes(2).withDCs(2).start());
+    }
+
+    @AfterClass
+    public static void after()
+    {
+        if (cluster != null)
+            cluster.close();
+    }
+    
+    @Before
+    public void enableHandoff()
+    {
+        cluster.get(1).nodetoolResult("enablehandoff");
+        cluster.get(2).nodetoolResult("enablehandoff");
+    }
+
+    @Test
+    public void testEnableHandoff()
+    {
+        cluster.get(node).nodetoolResult("statushandoff").asserts().success().stdoutContains("Hinted handoff is running");
```

Comment:
We could also use JMX to verify that hh is enabled:
```
assertTrue(cluster.get(node).callOnInstance(() -> StorageProxy.instance.getHintedHandoffEnabled()));
```
Also, we could test `enablehandoff` both when it is initially enabled and when it's initially disabled, wdyt?